src/cryptography/exceptions.py

```diff
@@ -53,5 +56,12 @@ class InvalidKey(Exception):
     pass
 
 
-class InvalidToken(Exception):
-    pass
+InvalidToken = utils.deprecated(
+    twofactor.InvalidToken,
+    __name__,
+    (
+        "The InvalidToken exception has moved to the "
+        "cryptography.hazmat.primitives.twofactor module"
+    ),
+    utils.DeprecatedIn08
```

Comment:
This should be DeprecatedIn09 (which means you'll need to add that to the utils module with value `PendingDeprecationWarning`)